Makefile

```diff
@@ -102,12 +102,3 @@ rundev:
 run:
 	docker-compose -f docker/docker-compose.yml up
 
-runlora:
```

Comment:
I removed this clutter because it is rarely launched and should be documented somewhere and not burden the main Makefile.